process faults {5: ({4, 0, 1}, 3622815730)}
network faults {6: [{0, 5}, {1}, {2}, {3}, {6, 4}]}
{5: ("696F57C8EF42A4DD51D5B8F9F45A8B44A98B739A96F5C4CDD1E2292E952E9E97", {5, 0, 6, 1, 2, 4, 3}), 6: ("18826EE78D644E8B2C0DC41E575EB0E8831BEA229D18C393625E3B5112CF5F73", {6, 5, 0, 4, 2, 1, 3}), 8: ("F7B80A28BC35F8E0B53846BDA7F15F4D2FDA56C3FE70BC19F66351FA8CA655C4", {4, 2, 0, 5, 3, 1, 6}), 9: ("BE58323BABA94BBCE762D9A561B78CF98D0E15CD27A8FB7E73A87715D573C904", {4, 2, 3, 6, 5, 1, 0}), 7: ("86AC6D9D3A0741F4E8F81767BFC631A4BC517C0D8734816F3604B517D39A04CF", {3, 5, 1, 4, 0, 6, 2}), 10: ("673DF99E220D25859F4EB21709C85D0F603A482E5C4ECF3C55123A73B5CF356F", {2, 3, 6, 1, 5, 4, 0})}
7
reason: all committed
done!
